Maker-Agents/ESMAgent/src/join.event.ts

```diff
@@ -39,12 +40,12 @@ const createFindingGenerator = (_alertID: string): FindingGenerator => {
     });
 };
```

Comment:
in filterLog function, why is there unused index and array parameters? how or where are they used?

Reply:
The `filter` argument for the `provideEventCheckerHandler` from the `forta-agent-tools` needs to be of that type. [Here](https://github.com/NethermindEth/general-agents-module#--event-checker-handler) is the description for it.

Also, for what I need, the `log` argument will suffice.